File: C#-Code/Demo.Hotel/handlers/connection.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Data.SqlClient;

namespace Demo.Hotel
{
    class connection
    {
        public static SqlConnection Con()
        {
            SqlConnection conection=new SqlConnection(@"Data Source=..\SQLEXPRESS;AttachDbFilename=|DataDirectory|\data\db_restorent.mdf;Integrated Security=True;User Instance=True");

            return conection; 
        }
    }
}

File: VB-Code/Form1/handlers/connection.vb
Imports System.Data.SqlClient
Public Class connection

    Public Function connction() As SqlConnection
        Dim con As SqlConnection
        con = New SqlConnection("Data Source=.\SQLEXPRESS;AttachDbFilename=|DataDirectory|\data\db_restorent.mdf;Integrated Security=True;User Instance=True")
        Return con

    End Function
End Class
